src/local/butler/common.py

```diff
@@ -378,6 +378,16 @@ def has_file_in_path(filename):
   return False
 
 
+def get_all_files(directory):
+  """Gets all files under a given directory."""
+  files = []
```

Comment:
lets make this a new variable `all_files` instead of overwritting the same variable in the loop I'm surprised this works

Reply:
Thanks for pointing out!